src/ocaml/preprocess/menhirLib.ml

```diff
@@ -3785,5 +3792,5 @@ module MakeEngineTable (T : TableFormat.TABLES) = struct
 end
 end
 module StaticVersion = struct
-let require_20201216 = ()
+let require_20210419 = ()
```

Comment:
There is no need to update `MenhirLib` and `parser_raw` with this PR.

Reply:
Ok, I added all the modified files. I'll revert them.